services/consensusalgo/leanhelixconsensus/test/harness.go

```diff
@@ -84,10 +100,15 @@ func (h *harness) expectConsensusContextRequestOrderingCommitteeNotCalled() {
 	h.consensusContext.When("RequestOrderingCommittee", mock.Any, mock.Any).Return(nil, nil).Times(0)
 }
 
-func (h *harness) expectConsensusContextRequestOrderingCommittee(leaderNodeIndex int) {
+func (h *harness) expectConsensusContextRequestOrderingCommittee(leaderNodeIndex int, times int) {
```

Comment:
`times` is never more than 1, why do you expose it?

Reply:
reverting... another relic of a previous version